kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -269,12 +273,21 @@
     float: left
     position: relative
     left: 10px
-    @media screen and (min-width: $portrait-breakpoint + 1)
+  @media screen and (min-width: $portrait-breakpoint + 1)
+    .searchbar
       font-size: 1em
       width: 100%
-    @media screen and (max-width: $portrait-breakpoint)
+  @media screen and (max-width: $portrait-breakpoint)
+    .create, #type-filter
+      box-sizing: border-box
+      width: 50%
+    .searchbar
       font-size: 0.8em
       width: 100%
       display: table-row
+    .table-username
+      display: none
+    .user-role
+      display: inline-block
```

Comment:
This is what makes the role jump to next line rather than have partially cut off background.